lib/opal/nodes/call.rb

```diff
@@ -226,11 +226,17 @@ def handle_special(&compile_default)
         file = compiler.file
         if arg.type == :str
           dir = File.dirname(file)
-          compiler.requires << Pathname(dir).join(arg.children[0]).cleanpath.to_s
+          mod_filename = Pathname(dir).join(arg.children[0]).cleanpath.to_s
+          compiler.requires << mod_filename
+        end
+        if compiler.es6_modules?
+          mod_filename = mod_filename.end_with?('.rb') ? mod_filename : mod_filename + '.rb'
+          push fragment("self.$require(#{Opal::Nodes::TopNode.module_name(mod_filename).inspect})")
```

Comment:
I don't see this method (`module_name`) in the `TopNode`. Was it moved to the compiler?